### ISSUE TITLE
[Accessibility]: aria-label of charts toolbar all have same ID

### ISSUE THREAD

**Issue by danielannan17** [carbon-design-system/carbon-charts#1838]
### Application/Team

API Connect

### Environment

Browser

### What happened? What did you expect to happen instead?

When you use multiple charts on the same page, you're given an accessibility violation due to all the chart toolbars having the same aria-label

![charts-toolbar-html](https://github.com/carbon-design-system/carbon-charts/assets/11006124/f15821ef-2cd8-449d-b130-c4755559974e)

![violation](https://github.com/carbon-design-system/carbon-charts/assets/11006124/5df8fd08-2cca-4c1a-889b-94610ffb88b0)

### What WCAG 2.1 checkpoint does the issue violate?

4.1.2

### Version

@carbon/charts-react@1.16.3

### What priority level would this issue be in your opinion?

P0

**Comment by tombrunet** [comment]
Adding some info, since this came up on accessibility office hours from another team. It looks like the aria-label is hardcoded and doesn't include the chart name or any identifying information from the chart:
https://github.com/carbon-design-system/carbon-charts/blob/08318c67c93dd53e31cb968976397f3068864e63/packages/core/src/components/axes/toolbar.ts#L42

It looks like someone tried to solve this using a random id, but then had to be pulled again because this solution doesn't account for the fact that these are human consumable string and not programmatic ids: (https://github.com/carbon-design-system/carbon-charts/commit/162e4104b6b1c83f65720f0203eb3c353774ef60)

Also related issue: https://github.com/carbon-design-system/carbon-charts/issues/1728

We need some way to have it be something like `${chart title} toolbar`, or provide the user for a way to override the toolbar title in the chart options.

**Comment by Mahalakshmivignesh** [comment]
@theiliad - Even after removing the aria-label, i still get the 'Multiple toolbar components do not have unique labels' issue. 
I am using the latest charts version 1.19.2 . This is the components HTML `<div class="cds--cc--toolbar" width="100%" height="100%" role="toolbar">`. 

Is there any additional changes to be done to make this error go?
